tests/rptest/services/utils.py

```diff
@@ -6,20 +6,46 @@
 from typing import Generator, Optional
 
 from rptest.clients.kubectl import KubectlTool, KubeNodeShell
-from rptest.services.cloud_broker import CloudBroker
+
+
+class Stopwatch():
+    def __init__(self) -> None:
```

Comment:
Nit: does this need `-> None`?  Folks know it's a constructor, right?  I just joined the party, may be annotating constructos is our conscious choice!

Reply:
It is not. I just have handy tool that does annotations automatically.